var/spack/repos/builtin/packages/httpie/package.py

```diff
@@ -26,12 +27,17 @@ class Httpie(PythonPackage):
         deprecated=True,
     )
 
+    depends_on("python", type=("build", "run"))
     depends_on("python@3.6:", when="@2.5:", type=("build", "run"))
-    depends_on("py-setuptools", type=("build", "run"))
     depends_on("py-charset-normalizer@2:", when="@2.6:", type=("build", "run"))
+    depends_on("py-colorama@0.2.4:", when="os=windows", type=("build", "run"))
     depends_on("py-defusedxml@0.6:", when="@2.5:", type=("build", "run"))
+    depends_on("py-importlib-metadata@1.4.0:", when="@3: ^python@:3.8", type=("build", "run"))
+    depends_on("py-multidict@4.7.0:", when="@3.2.1:", type=("build", "run"))
     depends_on("py-pygments@2.1.3:", type=("build", "run"))
     depends_on("py-pygments@2.5.2:", when="@2.5:", type=("build", "run"))
+    depends_on("py-requests-toolbelt@0.9.1:", when="@2.5:", type=("build", "run"))
     depends_on("py-requests@2.11:", type=("build", "run"))
     depends_on("py-requests@2.22:+socks", when="@2.5:", type=("build", "run"))
-    depends_on("py-requests-toolbelt@0.9.1:", when="@2.5:", type=("build", "run"))
+    depends_on("py-rich@9.10.0:", when="@3.2.1:", type=("build", "run"))
+    depends_on("py-setuptools", type=("build", "run"))
```

Comment:
It's helpful to sort these in the same order as in `setup.py` so I can quickly check that they match

Reply:
Good idea! Done.